src/contacts/components/ContactPage.js

```diff
@@ -370,12 +372,19 @@ class ContactPage extends Component<Props, State> {
       isSaveClicked,
       isSaving,
       match: {params: {contactId}},
+      userActiveServiceUnit,
       usersPermissions,
     } = this.props;
     const {activeTab, isRestoreModalOpen} = this.state;
 
     const nameInfo = getContactFullName(contact);
 
+    const isServiceUnitSameAsActiveServiceUnit = () => {
+      if (userActiveServiceUnit && contact.service_unit) {
```

Comment:
Can `userActiveServiceUnit` be undefined in some situation? When should that happen? If `userActiveServiceUnit` cannot be undefined, this check can be simplified with [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).